src/cliente/js/funciones.js

```diff
@@ -32,4 +32,30 @@ function montarMenu(url, vista) {
             return true
         }
     )
+}
+
+function redirigir(){
+    var url = extraerCookie("Redirect")
+    console.log(url)
+    if (url !== null){
+        url = decodeURIComponent(url.split("=")[1])
+        console.log(url)
+        url = url.split("/")[1]
+        console.log(url)
+        cargarVista(url)
+        borrarCookie("Redirect")
```

Comment:
Del bug que te he comentado: la función tiene que devolver `true` si si que se ha hecho la redirección, de no hacerlo `cargarVista("portada")` en el init se ejecuta siempre.

```suggestion
        borrarCookie("Redirect")
        return true
```
PD: quita los console.log